packages/wallets/src/components/WalletListHeader/WalletListHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Trans } from 'react-i18next';
+import { useActiveWalletAccount, useWalletAccountsList } from '@deriv/api-v2';
+import useDevice from '../../hooks/useDevice';
+import useWalletAccountSwitcher from '../../hooks/useWalletAccountSwitcher';
+import { WalletText } from '../Base';
+import './WalletListHeader.scss';
+
+const WalletListHeader: React.FC = () => {
+    const { isMobile } = useDevice();
+    const { data: wallets } = useWalletAccountsList();
+    const { data: activeWallet } = useActiveWalletAccount();
```

Comment:
Is it not needed to have the state for when this data is being loaded?

Reply:
Not sure if we'd need a loading state for the switcher itself. Need confirmation regarding this @nijil-deriv. But the handleToggle itself already checks if demoAccount and firstRealAccount exists to ensure values are not undefined

https://github.com/binary-com/deriv-app/assets/142860499/f09099e0-67f8-4451-add3-3079e83c569c

https://github.com/binary-com/deriv-app/assets/142860499/3672a3b2-468b-40ca-ba73-8e71a8437004

Reply:
The second one resembles more like Traders Hub in production. I will go for the second one @rostislav-deriv. Having a loading animation on a switcher is abit weird to be honest.

Reply:
Can we remove the delay in toggle switching between Real and Demo? I think the switch should be instant and not depend on the loading state. If there's an error and the account was not switched for some reason, the toggle can then go back to it's previous state.

Reply:
Updated change as per discussion: 

https://github.com/binary-com/deriv-app/assets/142860499/4971d15b-6527-487f-865f-baa68c7b0792